### PR TITLE
Mention complex number support in README.

### PR DESCRIPTION
**Pull request by vincentqb**
* Mention standard complex number in README, with functionals available.
* Update formatting for `torch.hann_window`

### CODE REVIEW

**Comment by vincentqb** [comment]
@cpuhrsch @jamarshon -- thoughts?

File: README.md [hunk 1]
```diff
@@ -127,6 +127,8 @@ Transforms expect the following dimensions.
 * `MuLawDecode`: (channel, time) -> (channel, time)
 * `Resample`: (channel, time) -> (channel, time)
 
+Complex numbers are supported via tensors of dimension (..., 2), and provide `magphase` to convert such a tensor into its magnitude and phase, and also `complex_norm` and `angle`.
+
```
Comment (jamarshon) [inline README.md]: maybe "its magnitude through `complex_norm` and phase through `angle`."

**Comment by jamarshon** [comment]
Yes I think this is a good idea to mention the dimensions of complex tensors until we move to a better solution (dtype, etc.) https://github.com/pytorch/audio/pull/131#issuecomment-508853162

File: README.md (hunk 1 above)
Comment (vincentqb) [reply README.md]: I agree we can simplify this. Let's only talk about `complex_norm` and `angle` then. How does that sound?

**Pull request merged**